docs/index.rst

```diff
@@ -13,44 +14,87 @@ Tidy3D Electromagnetic Solver
 .. image:: https://img.shields.io/github/actions/workflow/status/flexcompute/tidy3d/run_tests.yml?style=for-the-badge
 .. image:: https://img.shields.io/endpoint?url=https://gist.githubusercontent.com/daquinteroflex/4702549574741e87deaadba436218ebd/raw/tidy3d_extension.json
 .. image:: https://img.shields.io/github/license/flexcompute/tidy3d?style=for-the-badge
-.. image:: https://img.shields.io/badge/Demo-Notebooks-8A2BE2?style=for-the-badge
+.. image:: https://img.shields.io/badge/Demo-Notebooks-9647AE?style=for-the-badge
     :target: https://github.com/flexcompute/tidy3d-notebooks
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg?style=for-the-badge
    :target: https://github.com/psf/black
 
 
-.. To do items:
-.. * open simple example in colab with API saved as environment variable and `!pip install tidy3d` in the first line.
-.. * toggle between command line - notebook / python instructions in section 1
 Tidy3D is a software package for solving extremely large electrodynamics problems using the finite-difference time-domain (FDTD) method. It can be controlled through either an `open source python package <https://github.com/flexcompute/tidy3d>`_ or a `web-based graphical user interface <https://tidy3d.simulation.cloud>`_.
 
-If you’d rather skip installation and run an example in one of our web-hosted notebooks, `click here <https://tidy3d.simulation.cloud/notebook?file=StartHere.ipynb>`_.
+Get Started
+===========
 
-.. `TODO: open example in colab <https://github.com/flexcompute/tidy3d>`_
+.. tabs::
+
+    .. group-tab:: On the Cloud |:cloud:|
+
+        If you’d rather skip installation and run an example in one of our web-hosted notebooks, `click here <https://tidy3d.simulation.cloud/notebook?file=StartHere.ipynb>`_.
+
+    .. group-tab:: Install on Linux |:penguin:| & MacOS |:apple:|
```

Comment:
I think this should be the default tab, most people probably just want to find this `pip install` command so I think it should be immediately visible.